packages/web/app/pages/[orgId]/view/subscription/manage.tsx

```diff
@@ -324,7 +306,3 @@ function ManageSubscriptionPage(): ReactElement {
     </>
   );
 }
-
-export const getServerSideProps = withSessionProtection();
-
-export default authenticated(ManageSubscriptionPage);
```

Comment:
@dotansimha this fix https://tortilla-hq.slack.com/archives/D02G230RP2N/p1676338023148399?thread_ts=1676337974.534549&cid=D02G230RP2N